src/radical/pilot/raptor/master.py

```diff
@@ -26,32 +26,21 @@ def out(msg):
 #
 class Master(rpu.Component):
 
-    # FIXME: instead of the input and output queue, use the default component
-    #        facilities
-
     # --------------------------------------------------------------------------
     #
     def __init__(self, cfg=None, backend='zmq'):
```

Comment:
Can you please document the allowed and required fields in *cfg*?

Reply:
The config is fully optional.